src/components/Product.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+//This is the component for the product gallery with props for image, pricing, title and sticker.
+function Product({sticker, imgtext, image, title, price}) {
+    return <>
+        <article>
+            <span>{sticker}</span>
+            <img alt={imgtext} src={image}/>
+            <p>{title}</p>
+            <h4>{price}</h4>
+        </article>
+
+    </>
+
```

Comment:
Hier heb je ook onnodig het fragment gebruikt terwijl het maar een enkel element is.
Je hebt de afbeelding apart een titel property meegegeven maar als je bij de property img de titel meegeeft bij de naam (alt={title}) komt de naam ook bij de foto te staan en hoef je het niet extra aan te roepen.